packages/experience/src/hooks/use-global-redirect-to.ts

```diff
@@ -4,17 +4,34 @@ import { useCallback, useContext } from 'react';
 import PageContext from '@/Providers/PageContextProvider/PageContext';
 import UserInteractionContext from '@/Providers/UserInteractionContextProvider/UserInteractionContext';
 
+type Options = {
+  /**
+   * Whether to clear the interaction context session storage before redirecting.
+   * Set to `false` if this redirection is not the final redirection in the sign-in process (not the sign-in successful redirection).
+   */
+  shouldClearInteractionContextSession?: boolean;
+  /**
+   * Whether to use `window.location.replace` instead of `window.location.assign` for the redirection.
+   * Use `false` if this is a 3rd-party URL redirection (social sign-in or single sign-on redirection) that should be added to the browser history.
+   */
+  isReplace?: boolean;
```

Comment:
Should also mention the default values of these two props.
```
Defaults to: true
```

Reply:
updated.